system/Helpers/url_helper.php

```diff
@@ -447,9 +447,9 @@ function safe_mailto(string $email, string $title = '', $attributes = ''): strin
 		$output = '<script type="text/javascript">'
 				. 'var l=new Array();';
 
-		for ($i = 0, $c = count($x); $i < $c; $i ++)
+		foreach ($x as $i => $x)
```

Comment:
Is this not good practice, overwriting the original?

Reply:
The singularizing process only works in rector if the loop variable is a plural. I can manually change if that needed. Actually, the `$x` still works. Do you have suggestion of new variable of value var ?

Reply:
Ok. I always thought this type of assignment will overwrite the original. But maybe we can use another letter for the assignment, to make it less confusing.

Reply:
I updated to `$value`